MobileBuy/sample/src/main/java/com/shopify/sample/Constant.java

```diff
@@ -0,0 +1,7 @@
+package com.shopify.sample;
+
+public interface Constant {
+
+  int PAGE_SIZE = 10;
+  int THRESHOLD = 10;
```

Comment:
nit: move this consts to BasePaginatedListViewModel

Reply:
Threshold is the number of items from the bottom of the list that if the list is scrolled to them, it will trigger pagination.

Reply:
I know :) but the name is confusing what I meant is let's give it better naming